### PR TITLE
added few channels

### LINKED ISSUE #4205: Asianet Middle East
**Channel Name:** Asianet Middle East
**Country:** IN
**Language:** Malayalam
**Website:** xxx
**Stream URL (optional):** https://mhdtvworld.xyz/068c8e84-7457-414f-8178-ee34366fe841
**Notes (optional):** xxx

### LINKED ISSUE #4246: Sana TV
**Channel Name:** Sana TV
**Country:** IN
**Language:** Tamil
**Website:** xxx
**Stream URL (optional):** http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
**Notes (optional):** xxx

### LINKED ISSUE #4247: Peppers
**Channel Name:** Peppers
**Country:** IN
**Language:** Tamil
**Website:** xxx
**Stream URL (optional):** http://tvflix02.ddns.net/MALAYALAM__ASIANET_MIDDLE_EAST/2021/08/23/19/21/22-10000.ts?token=test
**Notes (optional):** xxx

### LINKED ISSUE #4248: Isai Aruvi
**Channel Name:** Isai Aruvi
**Country:** IN
**Language:** Tamil
**Website:** xxx
**Stream URL (optional):**http://103.199.161.254/Content/isaiaruvi/Live/Channel(IsaiAruvi)/Stream(01)/index.m3u8
**Notes (optional):** xxx

### LINKED ISSUE #4249: Sana Plus HD
**Channel Name:** Sana Plus HD
**Country:** IN
**Language:** Tamil
**Website:** xxx
**Stream URL (optional):** http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
**Notes (optional):** xxx

### LINKED ISSUE #4250: Yupp Thirai
**Channel Name:** Yupp Thirai
**Country:** IN
**Language:** Tamil
**Website:** xxx
**Stream URL (optional):** http://119.81.82.28/encoded/yuppthirai_800/chunklist.m3u8
**Notes (optional):** xxx

### PR DESCRIPTION
**Pull request by Niyasmkkv**
Closes #4250
Closes #4249 
Closes #4248
Closes #4247 
Closes #4246 
Closes #4205

### CODE REVIEW

**Comment by Niyasmkkv** [comment]
@LaneSh4d0w Help me to fix this errors.

File: channels/in.m3u [hunk 1]
```diff
@@ -185,6 +187,8 @@ https://indiatodaylive.akamaized.net/hls/live/2014320/indiatoday/indiatodaylive/
 https://live-indiatvnews.akamaized.net/indiatv-origin/ITV_1_1@199237/playlist.m3u8
 #EXTINF:-1 tvg-id="IndiaTV.in" tvg-name="India TV" tvg-country="IN" tvg-language="Hindi" tvg-logo="https://i.imgur.com/IYXrLPs.png" group-title="",India TV (480p)
 https://live-indiatvnews.akamaized.net/indiatv-origin/liveabr/playlist.m3u8
+#EXTINF:-1 tvg-id="IsaiAruvi.in" tvg-name="Isai Aruvi" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/radWIeb.png" group-title="Music",Isai Aruvi (576p)
+http://103.199.161.254/Content/isaiaruvi/Live/Channel(IsaiAruvi)/Stream(01)/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:191]:
```suggestion
http://103.199.161.254/Content/isaiaruvi/Live/Channel(IsaiAruvi)/playlist.m3u8
```

Adaptive link

File: channels/in.m3u [hunk 2]
```diff
@@ -70,6 +70,8 @@ https://live.hungama.com/linear/artist-aloud/playlist.m3u8
 http://ashrafichannel.livebox.co.in/ashrafivhannelhls/live.m3u8
 #EXTINF:-1 tvg-id="AsianetNews.in" tvg-name="Asianet News" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/89LBgkl.png" group-title="News",Asianet News (576p)
 http://103.199.161.254/Content/asianetnews/Live/Channel(Asianetnews)/index.m3u8
+#EXTINF:-1 tvg-id="AsianetMiddleEast.in" tvg-name="Asianet Middle East" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/du9iZ0s.png" group-title="Entertainment",Asianet Middle East (576p)
+http://tvflix02.ddns.net/MALAYALAM__ASIANET_MIDDLE_EAST/mono.m3u8?token=test
```
Comment (LaneSh4d0w) [inline channels/in.m3u:74]: Watch out, Flussonic server, might get out if "test" token gets off.

File: channels/in.m3u [hunk 3]
```diff
@@ -361,6 +365,8 @@ https://news18india-lh.akamaihd.net/i/news18india_1@174951/index_5_av-p.m3u8
 https://cdn.smartstream.video/smartstream-us/otvnewmbr/otvnewmbr.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="OneTv.in" tvg-name="One TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="" group-title="Entertainment",One TV
 http://137.59.86.218:1935/live/onetv/playlist.m3u8
+#EXTINF:-1 tvg-id="Peppers.in" tvg-name="Peppers" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/xgEQiso.png" group-title="Music",Peppers
+http://103.199.160.85/Content/peppers/Live/Channel(Peppers)/Stream(01)/index.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:369]:
```suggestion
http://103.199.160.85/Content/peppers/Live/Channel(Peppers)/playlist.m3u8
```

Adaptive link

File: channels/in.m3u (hunk 1 above)
Comment (Niyasmkkv) [reply channels/in.m3u:191]: Suggested link is not working.

File: channels/in.m3u [hunk 4]
```diff
@@ -411,6 +417,10 @@ http://cdn.clive.in:1935/sadhnabhakti/sadhnabhakti.stream_HDp/playlist.m3u8
 http://cdn.clive.in:1935/sadhnaplus/sadhnaplus.stream_HDp/media.m3u8
 #EXTINF:-1 tvg-id="SafariTV.in" tvg-name="Safari TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/LOepeP3.jpg" group-title="",Safari TV (480p)
 https://j78dp346yq5r-hls-live.5centscdn.com/safari/live.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="SanaTV.in" tvg-name="Sana TV" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Entertainment",Sana TV
+http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
+#EXTINF:-1 tvg-id="SanaPlus.in" tvg-name="Sana Plus" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Music",Sana Plus
+http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:423]: Umm... I see the same links right there.

File: channels/in.m3u [hunk 5]
```diff
@@ -411,6 +417,10 @@ http://cdn.clive.in:1935/sadhnabhakti/sadhnabhakti.stream_HDp/playlist.m3u8
 http://cdn.clive.in:1935/sadhnaplus/sadhnaplus.stream_HDp/media.m3u8
 #EXTINF:-1 tvg-id="SafariTV.in" tvg-name="Safari TV" tvg-country="IN" tvg-language="Malayalam" tvg-logo="https://i.imgur.com/LOepeP3.jpg" group-title="",Safari TV (480p)
 https://j78dp346yq5r-hls-live.5centscdn.com/safari/live.stream/playlist.m3u8
+#EXTINF:-1 tvg-id="SanaTV.in" tvg-name="Sana TV" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Entertainment",Sana TV
+http://media.7starcloud.com:1935/live/sanatv2/chunklist_w1266678806.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:421]:
```suggestion
http://media.7starcloud.com:1935/live/sanatv2/playlist.m3u8
```

Adaptive link

File: channels/in.m3u (hunk 2 above)
Comment (Niyasmkkv) [reply channels/in.m3u:74]: Any ways to fix that?

File: channels/in.m3u [hunk 6]
```diff
@@ -520,6 +530,8 @@ http://vasanth.live.cdn.bitgravity.com/vasanth/secure/live/feed03?bgsecuredir=1&
 http://95.216.167.183:5080/LiveApp/streams/443106610169904881506470.m3u8
 #EXTINF:-1 tvg-id="Velicham.in" tvg-name="Velicham" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/YeR3mm9.png" group-title="Entertainment",Velicham (360p) [Not 24/7]
 https://rtmp.smartstream.video/velichamtv/velichamtv/playlist.m3u8
+#EXTINF:-1 tvg-id="YuppThirai" tvg-name="YuppThirai" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Movies",Yupp Thirai
```
Comment (LaneSh4d0w) [inline channels/in.m3u:533]:
```suggestion
#EXTINF:-1 tvg-id="YuppThirai.in" tvg-name="Yupp Thirai" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Movies",Yupp Thirai 
```

File: channels/in.m3u (hunk 4 above)
Comment (Niyasmkkv) [reply channels/in.m3u:423]: http://hdserver.7starcloud.com:1935/sanatv/sanatv/chunklist_w1434725736.m3u8

File: channels/in.m3u (hunk 5 above)
Comment (Niyasmkkv) [reply channels/in.m3u:421]: Anything would do. Have no clue about adaptive links, Found these links from an app.

File: channels/in.m3u (hunk 4 above)
Comment (Niyasmkkv) [reply channels/in.m3u:423]: @LaneSh4d0w Here is the link for Sana Tv,
http://hdserver.7starcloud.com:1935/sanatv/sanatv/chunklist_w1434725736.m3u8

File: channels/in.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:191]: Yeah my bad, fixed it back on edition

File: channels/in.m3u (hunk 2 above)
Comment (LaneSh4d0w) [reply channels/in.m3u:74]: Apart from finding a new link, I don't quite know lol

File: channels/in.m3u (hunk 4 above)
Comment (LaneSh4d0w) [reply channels/in.m3u]: Changed.

**Review by Niyasmkkv — COMMENTED** [review]
We can close these issues now ryt?

File: channels/in.m3u [hunk 7]
```diff
@@ -520,6 +530,8 @@ http://vasanth.live.cdn.bitgravity.com/vasanth/secure/live/feed03?bgsecuredir=1&
 http://95.216.167.183:5080/LiveApp/streams/443106610169904881506470.m3u8
 #EXTINF:-1 tvg-id="Velicham.in" tvg-name="Velicham" tvg-country="IN" tvg-language="Tamil" tvg-logo="https://i.imgur.com/YeR3mm9.png" group-title="Entertainment",Velicham (360p) [Not 24/7]
 https://rtmp.smartstream.video/velichamtv/velichamtv/playlist.m3u8
+#EXTINF:-1 tvg-id="YuppThirai.in" tvg-name="Yupp Thirai" tvg-country="IN" tvg-language="Tamil" tvg-logo="" group-title="Movies",Yupp Thirai
+http://119.81.82.28/encoded/yuppthirai_800/chunklist.m3u8
```
Comment (LaneSh4d0w) [inline channels/in.m3u:534]:
```suggestion
http://119.81.82.28/encoded/yuppthirai_800/playlist.m3u8
```

**Review by BellezaEmporium — APPROVED** [review]

**Pull request merged**